proto/cosmos/group/v1beta1/types.proto

```diff
@@ -162,98 +159,99 @@ message Proposal {
   // When a decision policy is changed, existing proposals from previous policy versions will become invalid.
   uint64 group_policy_version = 7;
 
-  // Status defines proposal statuses.
-  enum Status {
-    option (gogoproto.goproto_enum_prefix) = false;
+  // status represents the high level position in the life cycle of the proposal. Initial value is Submitted.
+  ProposalStatus status = 8;
 
-    // An empty value is invalid and not allowed.
-    STATUS_UNSPECIFIED = 0 [(gogoproto.enumvalue_customname) = "ProposalStatusInvalid"];
+  // result is the final result based on the votes and election rule. Initial value is unfinalized.
+  // The result is persisted so that clients can always rely on this state and not have to replicate the logic.
+  ProposalResult result = 9;
 
-    // Initial status of a proposal when persisted.
-    STATUS_SUBMITTED = 1 [(gogoproto.enumvalue_customname) = "ProposalStatusSubmitted"];
+  // final_tally_result contains the sums of all weighted votes for this
+  // proposal for each vote option, after tallying.
+  TallyResult final_tally_result = 10 [(gogoproto.nullable) = false];
 
-    // Final status of a proposal when the final tally was executed.
-    STATUS_CLOSED = 2 [(gogoproto.enumvalue_customname) = "ProposalStatusClosed"];
+  // timeout is the timestamp of the block where the proposal execution times out. Header times of the votes and
+  // execution messages must be before this end time to be included in the election. After the timeout timestamp the
+  // proposal can not be executed anymore and should be considered pending delete.
+  google.protobuf.Timestamp timeout = 11 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
```

Comment:
This is the same as `voting_end_time` within gov right?

Reply:
It seems to me this `timeout` is the max of both `voting_end_time` and execution_end_time

Reply:
if we eventually get rid of the ability to execute proposals on demand as described in https://github.com/cosmos/cosmos-sdk/issues/11092#issuecomment-1027790198, then I guess we should eventually rename this to `voting_end_time` too but seems fine to keep it as is for now

Reply:
Since we're not changing the proposal execution model, I propose to keep it as-is.

Reply:
I think the comment here is still not clear we have two windows: The election window from submission time to the end of voting time where votes are accepted. Then we have the execution window from voting end time to timeout where `MsgExec` can be submitted in the case the proposal passes

Reply:
I don't think we have 2 windows. We just have one window for execution, which is this `timeout`. And obviously voting needs to pass before we do execution, but there's no window for voting only.

I can update the comment.